### ISSUE TITLE
Ability to add items above and below timeline - similar to Visio

### ISSUE THREAD

**Issue by stevehutchins** [visjs/vis-timeline#769]
Hi,
I am trying to move a bunch of Visio users across to vis.js timeline. The last sticking point is that they want to be able to add items above and below the timeline to reduce the width of the timeline whilst being able to see as many of the items as possible.
Is this possible?
Regards Steve

**Comment by yotamberk** [comment]
Can you add an illustration of what you mean?

**Comment by stevehutchins** [comment]
Hi,
I have added an image to the ticket. Thanks for taking a look at it.

Cheers
Steve

**Comment by yotamberk** [comment]
I don't see an image

**Comment by stevehutchins** [comment]
I can see it in the bottom of the ticket.
Here is the url from the ticket:

https://user-images.githubusercontent.com/13477288/98183805-d461af00-1f6d-11eb-9f1a-bb3bc132e0a1.png

Cheers
Steve

**Comment by yotamberk** [comment]
I don't think this library is fit for you if this is the requirement. I don't see anything close to this feature ever being implemented.
If this requirement is a must, I think you should look for a different alternative.
https://openbase.io/packages/top-react-timeline-libraries

**Comment by stevehutchins** [comment]
That's a shame. Is this because it is too hard to implement?
The vis.js library is a great tool.

Cheers
Steve

**Comment by yotamberk** [comment]
It requires another timeline

**Comment by smartsl** [comment]
I have a similar request: a group can be set to be upside or downside as wish.

Sadly it seems no one will dive into this.

**Comment by kemisstep** [comment]
+1

**Comment by papandadj** [comment]
+1
